.github/workflows/test-hosted-runners.yml

```diff
@@ -51,3 +51,7 @@ jobs:
     - name: Check for absence of symlinks
       run: if [ $(find artifact2 -type l | wc -l) != 0 ]; then echo "Symlinks found"; exit 1; fi
       shell: bash
+
+    - name: Check for absence of hidden files
```

Comment:
https://github.com/actions/upload-pages-artifact/blob/2eb9582f11ccb71dd662c5c63ad5b3c6d53cc27e/script/new-artifact.sh#L10

addtional change required ^ to generate hidden file, otherwise it test nothing :)